haascli/config.py

```diff
@@ -25,16 +28,16 @@ class HaasConfigurationKey(enum.Enum):
 class HaaSConfiguration(object):
     @staticmethod
     def load(config_path):
-        debug("Loading config from {}", config_path)
+        logger.debug("Loading config from {}".format(config_path))
         with open(config_path, 'r') as f:
             config_raw = yaml.load(f)
             config = {HaasConfigurationKey[k.upper()]: v for k, v in config_raw.items()}
-            debug(config)
+            logger.debug(config)
             return HaaSConfiguration(config)
 
     @staticmethod
     def dump(config, config_path):
-        debug("Writing config to {}", config_path)
+        logger.debug("Writing config to {}", config_path)
```

Comment:
This line is different from the others, which call the string.format method. The logging routines take arguments from the format string, ie: logger.debug("name: %s", name). It does not work with the {} syntax. I think is it cleaner to use the old style % format descriptors and send arguments to logging. How this is fine. Also, because this was not noticed, config is probably missing a few useful test cases.